### DISCUSSION TITLE
Carousel

### DISCUSSION THREAD

**Discussion by bramkragten** [home-assistant/frontend#3898] [Dashboard]
I would love to have a carousel to slide through multiple child cards.

Something like https://github.com/alex-saunders/sliding-pages

![image](https://camo.githubusercontent.com/2df97689a0d2be29796f9371de620b31653c755d/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78554137615377706535346d7048487a496b2f67697068792e676966)

**Comment by bramkragten** [comment]
Created this custom card:
https://community.home-assistant.io/t/lovelace-swiper-card/72447

**Comment by iantrich** [comment]
@bramkragten oooo I want this in official!

**Comment by zsarnett** [comment]
@bramkragten after the feature freeze. You should add this to core

**Comment by iantrich** [comment]
@bramkragten, do you think this should still come to core?

**Comment by bramkragten** [comment]
Yes!

**Comment by iantrich** [comment]
Well hurry up then! 😜

**Reply by KTibow** [reply]
How hard would this be to implement?

**Reply by zsarnett** [reply]
Its already built in custom. Just needs to be added to core @KTibow

**Reply by KTibow** [reply]
I mean could you just copy and paste the whole custom repo and change a couple lines, or rewrite from scratch?

**Reply by zsarnett** [reply]
I mean it would probably be best to rewrite some but it would be a lot of copy paste. I think
